scripts/seal_tests/custom_tests/batch_add_bytecode.txt

```diff
@@ -0,0 +1,4 @@
+3 1 a 1 5 1 ARR_IN
```

Comment:
should the bytecode be placed in `scripts/seal_tests/tests`? 

we also don't need to commit the bytecode!

Reply:
This was a separate folder for the handmade bytecode. I'll remove this from the commit.